pkg/kubelet/dockertools/manager.go

```diff
@@ -61,6 +61,8 @@ const (
 
 	MinimumDockerAPIVersion = "1.18"
 
+	MinimumDockerAPIVersion121 = "1.21"
```

Comment:
This name isn't very good. `MinimumDockerAPIVersion` is used by the Kubelet to ensure the node has at least the oldest Docker version still supported.

I would probably call this something like `minimumDockerAPIVersionForParallelPulls` (note the lowercasing - I don't think it needs to be exported)